### PR TITLE
Hrt 54 login page

### PR DESCRIPTION
**Pull request by gargi-bansal**
### Purpose
Created UI for login page so that users can log in to their accounts
[HRT-54](https://softeng761team5.atlassian.net/browse/HRT-54)

### Approach

Allows users to enter their username/email and password and access their charts

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR involve a UI change?

- [x] Yes
- [ ] No

#### Screenshot(s) of UI before this PR
N/A

#### Screenshot(s) of UI after this PR
Screen Shot 2020-09-26 at 4 06 07 PM

### CODE REVIEW

**Review by mitchfaulcon — DISMISSED** [review]
lgtm

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              <Typography className="Login-Label-Text">Password</Typography>
+              <Input className="Login-Input" name="password" type="password" />
+            </div>
+          </div>
+          <Button
+            type="primary"
+            className="Login-Button"
+            onClick={onConfirmClick}
+          >
+            <Typography className="Button-Font">Login</Typography>
```
Comment (preetpatel) [inline src/pages/Login.tsx]: Did you miss adding this class name to the styling?

**Comment by PreetPatel** [comment]
ezgif com-video-to-gif

Will have to request changes because of the above behaviour :'(

**Review by preetpatel — CHANGES_REQUESTED** [review]
ezgif com-video-to-gif

This doesn't look good.... 🥶

**Comment by gargi-bansal** [comment]
> ezgif com-video-to-gif
> 
> This doesn't look good.... 🥶

@PreetPatel yea I know the resizability issues are a work in progress, they're in a few of the other pages so I'm gonna fix them together ;(

**Review by preetpatel — DISMISSED** [review]

File: src/styles/Login.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  text-align: center;
+}
+
+.Create-Account-Text {
+  text-align: center;
+  font-size: 15px;
+}
+.Link-Text {
+  margin-left: 5px;
+  color: #ff0059 !important;
```
Comment (dder725) [inline src/styles/Login.css]: Are !important properties needed here? The styling seems to be working for me without those.

Comment (gargi-bansal) [reply src/styles/Login.css]: I'll have a look again, they weren't working for me before

Comment (gargi-bansal) [reply src/styles/Login.css]: @dder725 fixed now! thanks for that

**Comment by dder725** [comment]
Looks great! I didn't catch any problems with responsiveness 💯

**Review by dder725 — APPROVED** [review]

**Comment by gargi-bansal** [comment]
> Looks great! I didn't catch any problems with responsiveness 💯

haha yea, I made the responsiveness fixes in this branch so I want it in as reference so I can fix the other pages <3

**Review by preetpatel — APPROVED** [review]

**Pull request merged**